package.json

```diff
@@ -75,7 +69,25 @@
     "start": "webpack-dev-server",
     "build": "webpack -p",
     "build:analyze": "webpack -p --env.analyze",
-    "test": "karma start",
+    "test": "jest",
     "lint": "eslint 'src/**/*.js'"
+  },
+  "jest": {
```

Comment:
I sorta like moving this to `jest.config.js` just to keep this file cleaner. But either is fine.